bpf/lib/conntrack.h

```diff
@@ -845,6 +846,7 @@ static __always_inline int ct_create4(const void *map_main,
 
 	entry.lb_loopback = ct_state->loopback;
 	entry.node_port = ct_state->node_port;
+	relax_verifier();
```

Comment:
Btw, do we have a cheaper call for relax_verifier() internally with less overhead e.g. jiffies64()?

Reply:
:thinking: What do you have in mind? We also want something that has zero arguments to minimize impact on complexity.